src/Camera_TEST.cc

```diff
@@ -105,6 +105,12 @@ sdf::ElementPtr CameraToSdf(const std::string &_type,
     << "            <fy>281</fy>"
     << "            <cx>162</cx>"
     << "            <cy>124</cy>"
+    << "            <p_fx>282</p_fx>"
+    << "            <p_fy>283</p_fy>"
+    << "            <p_cx>163</p_cx>"
+    << "            <p_cy>125</p_cy>"
+    << "            <tx>1</tx>"
+    << "            <ty>2</ty>"
```

Comment:
Can we check that the parsing / setting of values was successful here ?

Reply:
If I am not mistaken this is already done in the test here:
https://github.com/gazebosim/gz-sensors/blob/6b1acb92a156dfe6e846d0b2392234ce6cfb6a90/src/Camera_TEST.cc#L123-L130

`sdf::readString` will return False if parsing was unsuccessful so the test should fail if the test is unsuccessful. One change that can be made is to check for the errors like this, but it's a little redundant since the return bool already tells us if it failed or not

```
  sdf::Errors errors;
  if (!sdf::readString(stream.str(), sdfParsed, errors)){
    ASSERT_EQ(errors.size(), 0);
    return sdf::ElementPtr();
}
```

Reply:
this does check that it did not fail during parsing but doesn't confirm that the values are correct. I don't see a test where this is already done, so I'll see if there's an easy way to add this; if not we can merge as is

Reply:
I've added some tests for the new parameters in https://github.com/gazebosim/gz-sensors/pull/249/commits/9c8f4282c243c286c99d584c9c7902e3b3009a8b